src/index.html

```diff
@@ -8,9 +8,37 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
```

Comment:
Also I get 404 page when I press enter on your searches, you should fix action value
![image](https://user-images.githubusercontent.com/37161315/137089225-04d5c937-fa06-4b63-8fc4-bac9b0fefbb4.png)